tests/helpers/helpers.ts

```diff
@@ -66,6 +82,17 @@ export const createTilesetFileName = (): string => {
   return 'tileset.json';
 };
 
+export const createInvalidTileset = (): string => {
+  return 'invalidtileset.json';
```

Comment:
Change the file name to "invalidTileset.json"

Reply:
```suggestion
  return 'invalidTileset.json';
```

Of course, change also the name of the file itself